fast-markdown-mcp/src/fast_markdown_mcp/server.py

```
@@ -221,6 +242,205 @@ async def search_files(self, query: str) -> str:
+                file_id = md_file.stem
+                
+                # Ensure the document structure is loaded
+                if file_id not in self.structure_cache:
+                    await self.get_content(file_id)
+                
+                structure = self.structure_cache[file_id]
+                
+                # Get all sections from the document
+                sections = []
+                
+                def collect_sections(section_list):
```

Ruff (0.8.2)

Missing return type annotation for private function `collect_sections`

Add return type annotation: `None`

(ANN202)